.circleci/config.yml

```diff
@@ -153,6 +169,35 @@ jobs:
           path: /tmp/test-reports
       - store_artifacts:
           path: /tmp/test-reports
+  test-ui:
+    docker:
+      - image: circleci/node:10-browsers
+        environment:
+          JOBS: 2
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-deps-{{ .Branch }}-{{ checksum "ui/yarn.lock" }}
+            - v1-deps-{{ .Branch }}-
+            - v1-deps-
+      - run:
+          name: yarn install
+          command: cd ui && yarn install
+      - save_cache:
+          key: v1-deps-{{ .Branch }}-{{ checksum "ui/yarn.lock" }}
+          paths:
+            - ./ui/node_modules
+      - run: export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
Unlike Travis - commands in circleci are executed on separate shells.  The export here has no effect for subsequent steps.  You can either export everytime it's needed (preferred) or use `BASH_ENV` hack, e.g. https://discuss.circleci.com/t/how-to-add-a-path-to-path-in-circle-2-0/11554/4 .

Reply:
heh nice, well this was doing nothing then! I removed it.